graphene_django/utils/utils.py

```diff
@@ -145,3 +145,18 @@ def bypass_get_queryset(resolver):
     """
     resolver._bypass_get_queryset = True
     return resolver
+
+
+def __django_version():
+    from pkg_resources import get_distribution
```

Comment:
The usage of `pkg_resources` is deprecated: https://setuptools.pypa.io/en/latest/pkg_resources.html

How about using:

```python
from django import VERSION
print(VERSION)
# (4, 1, 11, 'final', 0)
```

instead?

Reply:
Looks like @sjdemartini beat me to it